docs/ops.html

```diff
@@ -3946,6 +3946,7 @@ <h3>Enter Migration Mode on the Brokers</h3>
   </p>
 
   <ul>
+    <li><a href="#brokerconfigs_broker.id">broker.id</a>: If <code>broker.id.generation.enable</code> is enabled (default is enabled) and <code>broker.id</code> is not set during migration, it will fail. Ensure <code>broker.id</code> is set to a non-negative integer.</li>
```

Comment:
Looks better!
Maybe we can just say: `Ensure <code>broker.id</code> is set to a non-negative integer even if <code>broker.id.generation.enable</code> is enabled (default is enabled). ` because:
1. we don't need to mention `during migration` since this is a migration section
2. We can combine these 2 sentences.

WDYT?

Reply:
Yes, I think your version is more concise, making it easier to read. 😄 
I will adopt it